app/views/pages/contributors.html.erb

```diff
@@ -93,4 +93,18 @@ We're always looking for ideas from everyone, including <strong>mental health ad
 			<i class="fa fa-github fa-inline fa-left"></i><%= link_to('gCrew', 'http://github.com/gcrew', class: "spacing") %>
 		</div>
 	</div>
+	<div class="table_row">
+		<div class="table_cell" style="width: 200px; padding-right: 40px; vertical-align: middle">
```

Comment:
You will need to add "padding-bottom: 40px;" to the previous contributor's table_cells (line 84 and line 87). Otherwise, looks good to me! Great job :)